src/169._Majority_Element.md

```diff
@@ -0,0 +1,68 @@
+##  Majority Element
+
+### Description
+
+```Python
+Given an array of size n, find the majority element. The majority element is the element that appears more than ⌊ n/2 ⌋ times.
+
+You may assume that the array is non-empty and the majority element always exist in the array.
+
+Example 1:
+
+    Input: [3,2,3]
+    Output: 3
+
+Example 2:
+
+    Input: [2,2,1,1,1,2,2]
+    Output: 2
+```
+
+### Python Solutions
+
+#### Approach 1: Dict(Hash Table)
+
+* Time complexity : O(n)
+* Space complexity : O(n)
+
+```Python
+class Solution:
+    def majorityElement(self, nums):
+        """
+        Given an array of size n, find the majority element. The majority element is the element that appears more than ⌊ n/2 ⌋ times.
+
+        Args:
+            nums(List[int]): the name of array of integers
+
+        Returns: 
+            the element that appears more than ⌊ n/2 ⌋ times(int)
+        """
+        dic = {}
+        for num in nums:
+            if num not in dic:
+                dic[num] = 1
+            if dic[num] > len(nums)//2:
```

Comment:
`len(nums)//2` -> `len(nums) // 2:`

Reply:
OK，I have corrected it just now. Much thanks for your careful review